### DISCUSSION TITLE
Jemalloc maybe?

### DISCUSSION THREAD

**Discussion by huwdp** [lightspark/lightspark#675] [Ideas]
I was wondering whether it would be a good idea to use jemalloc? It can be easily integrated into Lighspark by using the "-ljemalloc" argument in GCC. The machine would have to have the library also. https://stackoverflow.com/a/9524031

It might improve performance and memory usage, or not. Lightspark seems suited as it uses threads and lots of memory.

**Comment by huwdp** [comment]
Firefox, Valgrind (I think), etc use it.

**Reply by dbluelle** [reply]
I haven't looked into that, but it seems you can try to change the new/delete operators in memory_reporter (memory_support.h) to use jemalloc and test if it really improves performance.
